### PR TITLE
addes non prod icon and debug menu

### PR DESCRIPTION
**Pull request by James-Pickett**
adds different icon and debug menu when launcher is pointed at a non Kolide prod host.

macos:
image

### CODE REVIEW

File: cmd/launcher/desktop/desktop.go [hunk 1]
```diff
@@ -18,12 +19,35 @@ func RunDesktop(args []string) error {
 	go exitWhenParentGone()
 	go handleSignals()
 
+	flagset := flag.NewFlagSet("launcher desktop", flag.ExitOnError)
+	var (
+		flhostname = flagset.String(
```
Comment (directionless) [inline cmd/launcher/desktop/desktop.go:24]: Ha! I guess this _does_ need a mechanism of getting this information into desktop.

I think we really will need to tackle IPC. But in the meantime... I might have suggested environmental variables, I think they make for nicer process listings, but this is probably fine.

File: ee/desktop/runtime/runner_darwin.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
 		return fmt.Errorf("determining executable path: %w", err)
 	}
 
-	proc, err := runAsUser(uid, executablePath, "desktop")
+	proc, err := runAsUser(uid, executablePath, "desktop", fmt.Sprintf("--hostname=%s", r.hostname))
```
Comment (directionless) [inline ee/desktop/runtime/runner_darwin.go:50]:
```suggestion
	proc, err := runAsUser(uid, executablePath, "desktop", "--hostname", r.hostname)
```

I think this should work

File: ee/desktop/runtime/runner_windows.go [hunk 3]
```diff
@@ -49,7 +49,7 @@ func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
 		}
 		defer accessToken.Close()
 
-		proc, err := runWithAccessToken(accessToken, executablePath, "desktop")
+		proc, err := runWithAccessToken(accessToken, executablePath, "desktop", fmt.Sprintf("--hostname=%s", r.hostname))
```
Comment (directionless) [inline ee/desktop/runtime/runner_windows.go:52]: same

File: ee/desktop/runtime/runner_test.go [hunk 4]
```diff
@@ -92,7 +92,7 @@ func TestDesktopUserProcessRunner_Execute(t *testing.T) {
 
 			var logBytes threadSafeBuffer
 
-			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1)
+			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1, "some-where-over-the-rainbow.com")
```
Comment (directionless) [inline ee/desktop/runtime/runner_test.go:95]:
```suggestion
			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1, "some-where-over-the-rainbow.example.com")
```

I feel humorless, but....

File: cmd/launcher/desktop/desktop.go (hunk 1 above)
Comment (James-Pickett) [reply cmd/launcher/desktop/desktop.go:24]: agree need to tackle IPC

**Review by directionless — APPROVED** [review]

**Pull request merged**